.babelrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Isn't this a duplicate of `babel.config.js`?

Reply:
partially? I'm not sure if it is required or not by webpack